gapis/api/vulkan/command_buffer_rebuilder.go

```diff
@@ -725,6 +725,102 @@ func rebuildVkCmdDrawIndexedIndirect(
 	), nil
 }
 
+func rebuildVkCmdDrawIndirectCountKHR(
+	ctx context.Context,
+	cb CommandBuilder,
+	commandBuffer VkCommandBuffer,
+	r *api.GlobalState,
+	s *api.GlobalState,
+	d VkCmdDrawIndirectCountKHRArgsʳ) (func(), api.Cmd, error) {
+	if !GetState(s).Buffers().Contains(d.Buffer()) {
+		return nil, nil, fmt.Errorf("Cannot find Buffer %v", d.Buffer())
+	}
+	if !GetState(s).Buffers().Contains(d.CountBuffer()) {
+		return nil, nil, fmt.Errorf("Cannot find Buffer %v", d.Buffer())
```

Comment:
It might be helpful to say "Count Buffer" here and in each of the other functions, so that it's more clear which argument the error is for.

Reply:
Done